screenshots/build.js

```diff
@@ -0,0 +1,17 @@
+var page = require('webpage').create();
+page.viewportSize = {width: 1000, height: 1000};
+page.open('./test/perceptual/alert.html', function() {
+	var clipRect = page.evaluate(function () {
+		var screenshot = document.querySelector(".screenshot");
+		screenshot.style.display = 'inline-block';
+		return screenshot.getBoundingClientRect();
+	});
+	page.clipRect = {
+		top:    clipRect.top,
+		left:   clipRect.left,
+		width:  clipRect.width,
+		height: clipRect.height
+	};
+	page.render('screenshot.png');
```

Comment:
We copied most of this from vui-button... when we run `npm run screenshot` on the commandline it places this in the root of the repo (with `package.json`) and not the screenshot folder. The same happens in vui-button... is this expected/desired?